src/pages/v3/index.tsx

```
@@ -0,0 +1,101 @@
+                        </p>
+                        <div className={styles.buttons}>
+                            <Link
+                                className="button button--secondary button--lg lg:text-2xl lg:px-8 lg:py-4"
+                                to="/docs/3.x"
+                            >
+                                Check V3 Docs →
+                            </Link>
+                            <a
+                                href="https://stackblitz.com/~/github.com/zenstackhq/v3-doc-quick-start?file=zenstack%2fschema.zmodel&file=main.ts&view=editor&showSidebar=0&hideNavigation=1&hideExplorer=1"
+                                target="_blank"
+                                className="button button--outline button--lg border-solid lg:text-2xl lg:px-8 lg:py-4 ml-4 hover:text-gray-200 dark:hover:text-gray-600"
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)